dnf-behave-tests/features/steps/file.py

```diff
@@ -66,13 +66,21 @@ def step_delete_directory(context, dirpath):
 @behave.step('file "{filepath}" exists')
 def file_exists(context, filepath):
     full_path = prepend_installroot(context, filepath)
-    ensure_file_exists(full_path)
+    result = glob.glob(full_path)
+    if len(result) > 1:
+        raise AssertionError("File path %s matches multiple files: \n%s" % (filepath, '\n'.join(result)))
+    elif len(result) < 1:
+        raise AssertionError("File path %s doesn't match any file." % (filepath))
+    else:
+        ensure_file_exists(result[0])
```

Comment:
Is this call really needed if the glob matched the path? (If `ensure_file_exists()` called `os.path.isfile()` it would be an argument, but it just calls `os.path.exists()`)

Reply:
I was unsure about it as well. But you are right since it just checks whether it exists the result will be the same (it will still succeed even with directories). I have remove the call.